lib/app/pages/add_folder_words_page/add_folder_words_page.dart

```diff
@@ -147,13 +152,29 @@ class _BodyState extends State<_Body> {
             child: widget.wordsList.isNotEmpty ? AddedWordsGrid(wordsList: widget.wordsList) : null,
           ),
           CustomTextField(
+            prefix: Padding(
```

Comment:
the problem with prefix is that it shows only after the textfield is focused. There is an iconData inside the CustomTextField. I think you should refactor the code a bit to accept widget "leading" and remove the hard-coded Icon widget inside it

Reply:
Suggestion: to make it even better you can add an autofocus to this text field so when user enter the screen he will automatically have the input focused